content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md

```diff
@@ -19,9 +19,9 @@ PersistentVolume.
 
 ## Why change reclaim policy of a PersistentVolume
 
-`PersistentVolumes` can have various reclaim policies, including "Retain",
-"Recycle", and "Delete". For dynamically provisioned `PersistentVolumes`,
-the default reclaim policy is "Delete". This means that a dynamically provisioned
+`PersistentVolumes` can have various reclaim policies, including Retain,
+Recycle, and Delete. For dynamically provisioned `PersistentVolumes`,
+the default reclaim policy is Delete. This means that a dynamically provisioned
```

Comment:
This is the wrong change. Instead of removing double quote characters: <tt>"</tt>, the issue calls for removal of (some) backticks: <tt>`</tt>.

Reply:
done thanks